app/code/Magento/Catalog/Test/Mftf/Test/EndToEndB2CGuestUserTest.xml

```diff
@@ -17,6 +17,198 @@
             <description value="User browses catalog, searches for product, adds product to cart, adds product to wishlist, compares products, uses coupon code and checks out."/>
             <severity value="CRITICAL"/>
             <testCaseId value="MAGETWO-87435"/>
+            <group value="SearchEngineElasticsearch"/>
```

Comment:
Please, revert changes, files in the Catalog module are part of Magento core, not MSI project

Reply:
Reverted changes